spec/takeaway_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'takeaway'
+
+describe TakeAway do
+  let(:takeaway) { TakeAway.new }
```

Comment:
I may be wrong here please don't quote me on this. 

I think the doubles should only be used when you are depending on another class and it's methods. 

For example your Menu class would need a double here, just incase there is a mistake in the menu.rb file then it would cause an error here and it would be hard to figure out which file is causing that error.

Reply:
It's not a double, it's a stub, to be able to call takeaway.new easily